### PR TITLE
Minor fix in TwoPhaseFlowWithPP process

### PR DESCRIPTION
**Pull request by Yonghui56**
This pull request includes two parts:
1)  the pr adds the missing documents of the two benchmarks for TwoPhaseFlowWithPP process
2)  fix the typo, the second benchmark is "McWhorter problem".
.

### CODE REVIEW

File: web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
++++
+project = "Parabolic/TwoPhaseFlowPP"
```
Comment (wenqing) [inline web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md]: Missing a project file name hides the page on the  web site. Change it to  
project = "Parabolic/ThermalTwoPhaseFlowPP/HeatPipe/Twophase_HeatPipe_quad_curve.prj.prj"

File: web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+{{< data-link >}}
+
+## Problem description
+
+Two benchmarks are presented here which are dedicated to simulating the two-phase flow transport in porous media with the pessure-capillary pressure as the primary variables. For the detailed benchmark settings and results description, please refer to the attached document.
+
+## Results and evaluation
+
+See [this PDF](../main.pdf).
```
Comment (wenqing) [inline web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md]: Please change `PP` in the directory name to 'pp', i,e to
```
web/content/docs/benchmarks/two-phase-flow-pp
```
The capital letters block the file main.pdf to be read.

File: web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md (hunk 1 above)
Comment (wenqing) [reply web/content/docs/benchmarks/two-phase-flow-PP/TwoPhaseFlowPP.md]: You have two tests but only one project file is allowed in TwoPhaseFlowPP.md. In order to have links to all the two project files, you have to prepare two md files for the two project files, respectively.

**Comment by Yonghui56** [comment]
Actually, I don't know why the CTESTs are not updated, either.
see https://doxygen.opengeosys.org/ogs_ctest_prj__Parabolic__TwoPhaseFlowPP__McWorts.html

**Pull request merged**